### DISCUSSION TITLE
Code challenge - Collection and Study Mangement

### DISCUSSION THREAD

**Discussion by jhansidundu** [camicroscope/GSOC#23] [Q&A] [answered]
Hi @nanli-emory, As part of the code challenge, i created a simple web app to manage tasks. Its not a very complex app by any means. It has the following functionalities. 
- [x] Create tasks
- [x] Update tasks
- [x] Read tasks
- [ ] Delete tasks

**UI Screenshots**

1. ![Screenshot_20230318_103948](https://user-images.githubusercontent.com/116087257/226122590-ea55e068-5e8b-4876-acab-4b90737733f7.png)

1. ![Screenshot_20230318_103936](https://user-images.githubusercontent.com/116087257/226122591-b9ff9726-94c8-4c41-b9f8-ab40b2be54d1.png)

1. ![Screenshot_20230318_104232](https://user-images.githubusercontent.com/116087257/226122589-366d0a42-9b8c-41fe-9e93-3b44fb7ee14d.png)

1. ![Screenshot_20230318_104257](https://user-images.githubusercontent.com/116087257/226122583-fe02a38c-c61f-4b5f-91b4-032fdf17656d.png)

Users will have to sign up with an email. I used mongodb to store the data of users. It currently has two collectiosn. One to store user info and one to store task info. 

I deployed the frontend in netlify, and backend in render.  [Website](https://curious-genie-0968d8.netlify.app/)
**Please find the git repositories here** 
- [frontend](https://github.com/jhansidundu/task-manager-frontend)  
-  [backend](https://github.com/jhansidundu/task-manager-backend)  

Can you take a look at my project and provide feedback on it?

**Comment by nanli-emory** [comment, marked as answer]
hi @jhansidundu, sorry for the delay in responding. I tried your website. looks good. Have you submitted your proposal?

**Reply by shaiktahseen** [reply]
Hi camicroscope community, I'm Shaik Tahseen. I'm a beginner and have knowledge in Html, CSS, JavaScript & Flexboxes, and I'm curious to join GSOC 2023 this year.
I like to contribute Can anyone help me to know how to join Jenkins & how to contribute & get into camicroscope  as a contributor with basic skills?  I really like to learn, contribute, give my best & look for opportunities where I can upskill myself & also fulfill the goals of the organization.

If anyone let me know what would be really helpful.
I see the closing date for GSOC is 4th April 2023

Here are my bootcamp certifications.
HTML & CSS Essentials Bootcamp
https://verify.letsupgrade.in/certificate/LUEHTMLFEB123307

JavaScript Programming Essentials Bootcamp
https://verify.letsupgrade.in/certificate/LUEJSMAR123657

•	Email : shaiktahseen05@gmail.com
•	GitHub :  https://github.com/shaiktahseen
•	LinkedIn :  https://www.linkedin.com/in/shaik-tahseen-%F0%9F%87%AE%F0%9F%87%B3-39074b17a/
•	Twitter: https://twitter.com/shaiktahseen\_05

**Reply by nanli-emory** [reply]
Hi @shaiktahseen do you submit your project proposals? You have to submit your project proposals before the deadline.

**Reply by shaiktahseen** [reply]
No i havn't submitted my proposals. 

I dont know how to submit proposal i see last date to submit proposal 4th april 2023.
I'm a beginner I mean anyone can help let me know how to submit proposal what to write, what not to write, how to structure etc... that would be really helpful to me.

**Reply by rudrakshnalbalwar** [reply]
Hi caMicroscope community, @nanli-emory , I am Rudraksh Nalbalwar, I have a question ? Can I directly submit my proposal even though I haven't contributed on Github since the major parts of the bugs are fixed. My proposal is ready .

**Reply by rudrakshnalbalwar** [reply]
Also I have prior experience in HTML, CSS, Javascript, Bootstrap, ReactJs.

**Reply by shaiktahseen** [reply]
@rudrakshnalbalwar Bro Good to hear you are ready with proposal.
Can you let me know how to write proposal, how to prepare, what to write in format ?

**Reply by shaiktahseen** [reply]
@nanli-emory I have experience in in HTML, CSS, Javascript made portpolio website.

I like to contribute Can anyone help me to know how to join caMicroscope & how to contribute & get into camicroscope as a contributor with basic skills? 

I really like to contribute, learn give my best & look for opportunities where I can upskill myself & also fulfill the goals of the organization.

**Reply by rudrakshnalbalwar** [reply]
@nanli-emory  I am a very enthusiastic person also I am always ready to face new challenges as it helps me in upskilling myself, I am very eager to participate in GSoC'23 as this is my first time participation in open source projects, also I am ready to dedicate 5-6 hrs daily for the GSoC'23 .

**Reply by jhansidundu** [reply]
> hi @jhansidundu, sorry for the delay in responding. I tried your website. looks good. Have you submitted your proposal?

Hi @nanli-emory, I am really sorry for not replying sooner. I am glad that you liked the website. I haven't submitted my proposal. Are you still accepting the proposals? If yes, i will submit by tomorrow

**Comment by rudrakshnalbalwar** [comment]
It is given in the guide

**Reply by shaiktahseen** [reply]
@rudrakshnalbalwar can you once share your proposal document that would be really helpful to me to know how to write, what to write.

**Comment by nanli-emory** [comment]
The deadline is April 04th before the deadline you can submit endless times you want.

**Reply by jhansidundu** [reply]
okay, thankyou

**Reply by jhansidundu** [reply]
Hi @nanli-emory, i submitted a draft proposal. How can I share the proposal with you to get some suggestions. Thanks!
